views/index.ejs

```diff
@@ -44,8 +44,8 @@
             <i id="toggle-icon" class="fa fa-angle-right"></i>
           </button>
         </div>
-      </div>
-      <div class="right active" id="lobby-msg" data-joined="1">
+      </div> 
+      <div class="swipe-area right active" id="lobby-msg" data-joined="1">
```

Comment:
adding `swipe-area` class here adds swipe action in lobby-room, but this action is not working in any room other than lobby. You need to add this in `client.js` too.

Reply:
I have to made the required changes .